lib/graphs/digraph.nit

```diff
@@ -0,0 +1,561 @@
+# Implementations for representing directed graphs, also called digraphs.
+#
+# Currently, there are two classes
+module digraph
+
+# An arc of a digraph
+class Arc[V]
```

Comment:
`V` alone with no bound means `nullable Object`. Does an arc can have only one end?

I will go for a bounded type on Object:

~~~nit
class Arc[V: Object]
~~~